Nonprofit-Exchange-Hub/web-app
# 364/submit profile image

## Dev Summary

- Standardize CTA buttons per our design docs
- Use new standard CTA buttons
- Make step four form use form input element
- New profile type
- Update user profile after register (this won't fully work until we fix out email verification)




## Test Plan


repro steps:

1. go to `signup-citizen`
2. fill out the form, add an image in the `upload` part of the 4th step. Expect to see validation on extension and file size
3. watch the network call
4. For now, this causes an unauthorized response because the user hasn't yet confirmed their email



## Resources
